cmd/opm/index/add.go

```diff
@@ -108,9 +108,13 @@ func runIndexAddCmdFunc(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	skipTLS, err := cmd.Flags().GetBool("skip-tls")
-	if err != nil {
-		return err
+	var skipTLS *bool
+	if cmd.Flags().Changed("skip-tls") {
```

Comment:
Why is this necessary for this flag? Is this used to check when a child command has overridden the flag from a parent command?

Reply:
We have different default behavior with docker and podman for the tls verify. I'm not sure if we have any usage where we omit the flag because we expect that default behavior. skipping TLS by default isn't something we should encourage, this is mainly for development purposes. But enabling TLS by default means that any scripts that rely on docker enabling insecure pulls by default will fail because they lack the option.

So this was just a way to add a third "stick to the tool's default behavior" option

Reply:
IMO, we should make `SkipTLSVerify=false` the default across the board. I'm more concerned about this being understandable than breaking existing UX.

Reply:
This was a misunderstanding on my part about docker's flags, it checks tls by default for any remote repository. The pointer flag isn't necessary.